src/SuggestionsOverlay.js

```diff
@@ -82,7 +82,7 @@ class SuggestionsOverlay extends Component {
 
     return (
       <div
-        {...inline(style, { position, left, top })}
```

Comment:
the problem here is, that the `position: undefined` inline style overrides the default `position: 'absolute'`